src/services/UserService.ts

```diff
@@ -42,4 +44,15 @@ export class UserService {
   async findById(id: number): Promise<User | undefined> {
     return await this.userRepository.findOne({ where: { id } });
   }
+
+  async recoverPassword(email: string): Promise<void> {
+    const user: User = await this.userRepository
+      .createQueryBuilder('user')
+      .where('user.email = :email', { email })
+      .getOne();
+    if (user) {
+      const token = await jwtRecoverPassword.createToken(String(user.id));
```

Comment:
Se for usar essa classe pra tratar JWTs precisa alterar nos locais em que já está sendo utilizado. Se não fica despadronizado com o restante do projeto.